editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -1302,7 +1333,7 @@ describe('actions', () => {
 
         const canvasRoot = renderResult.renderedDOM.getByTestId('canvas-root')
 
-        await undoCheckerFn(renderResult, async () => {
+        await expectSingleUndoNSaves(renderResult, test.generatesUndoSteps ?? 2, async () => {
```

Comment:
it's a single _undo step_ and N saves! so the field should be renamed `test.generatesSaves` or `test.generatesSaveCount` or `test.triggeredSaveNTimes`